### PR TITLE
BaSP-TG-24: tests update delete admins

### PR DESCRIPTION
**Pull request by sofihoj**

### CODE REVIEW

File: src/tests/admins.test.js [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+    expect(response.body.error).toBeFalsy();
+    expect(response.body.message).toEqual(`Admin with id ${validId} updated successfully`);
+  });
+
+  test('Should not modify an admin because of missing required fields', async () => {
+    const response = await request(app).put(`/admins/${validId}`).send(incompleteAdmin);
+
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBe(true);
+    expect(response.body.data).toBe(undefined);
```
Comment (CristianBucci) [inline src/tests/admins.test.js:54]: You can test response.body.message toBeDefined!

File: src/tests/admins.test.js [hunk 2]
```diff
@@ -1,5 +1,109 @@
...
+  });
+
+  test('Should not modify an admin because of invalid body', async () => {
+    const response = await request(app).put(`/admins/${validId}`).send(wrongAdmin);
+
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBe(true);
+    expect(response.body.data).toBe(undefined);
```
Comment (CristianBucci) [inline src/tests/admins.test.js:62]: Same as line 54!

File: src/tests/admins.test.js [hunk 3]
```diff
@@ -1,5 +1,109 @@
...
+    expect(response.body.error).toBeFalsy();
+    expect(response.body.message).toEqual('Admin deleted successfully');
+  });
+
+  test('Should not delete an admin because of invalid id', async () => {
+    const response = await request(app).put(`/admins/${invalidId}`).send();
+
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBe(true);
+    expect(response.body.data).toBe(undefined);
```
Comment (CristianBucci) [inline src/tests/admins.test.js:98]: Same as line 54!

File: src/tests/admins.test.js (hunk 1 above)
Comment (sofihoj) [reply src/tests/admins.test.js:54]: Done, Thanks :)

File: src/tests/admins.test.js [hunk 4]
```diff
@@ -1,5 +1,112 @@
...
+    const response = await request(app).put(`/admins/${invalidId}`).send(mockedAdmin);
+
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBe(true);
+    expect(response.body.data).toBe(undefined);
+    expect(response.body.message).toEqual(`Admin with id ${invalidId} not found`);
+  });
+
+  test('Should fail to modify admin: id not found', async () => {
+    const response = await request(app).delete(`/admins/${idNotFound}`).send();
```
Comment (RamiroAmadeo) [inline src/tests/admins.test.js:77]: should be ".put"

Comment (sofihoj) [reply src/tests/admins.test.js:77]: You're right!

**Review by Alfondalix — DISMISSED** [review]

**Review by Alfondalix — APPROVED** [review]

**Pull request merged**